packages/dotcom-build-code-splitting/src/__test__/index.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { PageKitCodeSplittingPlugin } from '../index'
+import webpack from 'webpack'
+import path from 'path'
+
+describe('Code Splitting', () => {
+  it('chunk privacy modules', async () => {
+    await new Promise((resolve) =>
+      webpack(
+        {
+          mode: 'none',
+          entry: {
+            scripts: path.join(__dirname, '/__fixtures__', 'entry-point.js')
+          },
+          output: {
+            filename: '[name].js',
+            path: path.join(__dirname, '/tmp')
+          },
+          plugins: [new PageKitCodeSplittingPlugin()]
+        },
+        function (err, stats) {
```

Comment:
Is `stats` short for statistics here? It's unclear from its usage exactly what this is.

Reply:
yes sort of, it contains info regarding timings and file chunks as well as any errors

https://webpack.js.org/api/node/#stats-object